src/vs/workbench/contrib/terminal/browser/terminal.contribution.ts

```diff
@@ -175,6 +175,14 @@ registerSendSequenceKeybinding(String.fromCharCode('W'.charCodeAt(0) - 64), {
 	primary: KeyMod.CtrlCmd | KeyCode.Backspace,
 	mac: { primary: KeyMod.Alt | KeyCode.Backspace }
 });
+if (platform.isWindows) {
+	// Delete word left: ctrl+h
+	// Windows cmd.exe requires ^H to delete full word left
+	registerSendSequenceKeybinding(String.fromCharCode('H'.charCodeAt(0) - 64), {
+		when: ContextKeyExpr.equals(KEYBINDING_CONTEXT_TERMINAL_SHELL_TYPE_KEY, WindowsShellType.CommandPrompt),
+		primary: KeyMod.CtrlCmd | KeyCode.Backspace,
+	});
+}
```

Comment:
@TylerLeonhardt any idea why ctrl+H deletes a word left for me in both pwsh and windows powershell on windows terminal but in VS Code only works in pwsh (windows powershell will delete a _character_ left)?

Reply:
hmm what versions of PSReadLine are available in those?

```
gmo PSReadLine
```

Reply:
It says 2.0.0 for both in Windows PowerShell.

Reply:
What edit mode are you using?:

```pwsh
Get-PSReadLineOption | % EditMode
```

cc @daxian-dbw

Reply:
`Windows` on both, I wonder if Windows Terminal does something clever here, the keybindings don't show anything bound to ctrl+h:

```json
    "keybindings":
    [
        // Copy and paste are bound to Ctrl+Shift+C and Ctrl+Shift+V in your defaults.json.
        // These two lines additionally bind them to Ctrl+C and Ctrl+V.
        // To learn more about selection, visit https://aka.ms/terminal-selection
        { "command": {"action": "copy", "singleLine": false }, "keys": "ctrl+c" },
        { "command": "paste", "keys": "ctrl+v" },

        // Press Ctrl+Shift+F to open the search box
        { "command": "find", "keys": "ctrl+shift+f" },

        // Press Alt+Shift+D to open a new pane.
        // - "split": "auto" makes this pane open in the direction that provides the most surface area.
        // - "splitMode": "duplicate" makes the new pane use the focused pane's profile.
        // To learn more about panes, visit https://aka.ms/terminal-panes
        { "command": { "action": "splitPane", "split": "auto", "splitMode": "duplicate" }, "keys": "alt+shift+d" }
    ]
```

Reply:
Wait sorry, can you run this:
```
Get-Module PSReadLine | % PrivateData | % PSData
```

In Windows PowerShell, it won't tell you if it's a prerelease... only in pwsh will it tell you.

Reply:
On both:

```
Name                           Value
----                           -----
Prerelease                     beta2
```

Reply:
In Windows Terminal, `[Console]::ReadKey()` shows `Ctrl+Backspace` for <kbd>ctrl+h</kbd>, while on the legacy console host, it shows `Ctrl+H`.
<kbd>ctrl+Backspace</kbd> is bound to `BackwardKillWord` in Windows edit mode, then hence it deletes a word in Windows Terminal.

Reply:
Thanks @daxian-dbw, found the wt PR https://github.com/microsoft/terminal/pull/3935

@zadjii-msft I followed the code in https://github.com/microsoft/terminal/pull/3935 and end up hitting `INPUT_RECORD`s, is this support only possible once conpty gets https://github.com/microsoft/terminal/pull/6309 (which I'm guessing is 12 months or something)?